test/join.jl

```diff
@@ -578,4 +578,30 @@ module TestJoin
         @test eltypes(join(l, r, on=:b, kind=:outer, makeunique=true)) ==
             [Union{Int, Missing}, CS, Union{Int, Missing}]
     end
+
+    @testset "indicator columns" begin
+        outer_indicator = DataFrame(ID = [1, 2, 2, 3, 4],
+                                Name = ["John Doe", "Jane Doe", "Jane Doe", "Joe Blogs", missing],
+                                Job = ["Lawyer", "Doctor", "Florist", missing, "Farmer"],
+                                _merge = categorical(["both", "both", "both", "left_only", "right_only"])
+                                );
```

Comment:
Close this on the previous line, no semicolon, and align with first argument (not with parenthesis). Same below.